tests/tests.go

```diff
@@ -315,6 +306,40 @@ func changeReplicas(managerName string, numOfReplica int32) error {
 	return nil
 }
 
+func deletePodsFromDeployment(deploymentName string) error {
```

Comment:
maybe `restartPodsFromDeployment` is more exact here?

Reply:
Done